dm_regional_app/utils.py

```diff
@@ -33,12 +32,29 @@ def __init__(self, *args, **kwargs):
 
     def parse_object(self, obj):
         obj = self.parse_dates(obj)
+        # Check for Series
         if "__type__" in obj and obj["__type__"] == "pd.Series":
-            if obj.get("is_multiindex", False):
-                index = pd.MultiIndex.from_tuples(obj["index"])
+            if obj.get("is_multiindex") == True:
```

Comment:
You can do 
```suggestion
            if obj.get("is_multiindex"):
```
but up to you on what you think is more readable!